.devcontainer/scripts/postStartCommand.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo "################################"
+echo "#  Creating your Vehicle App   #"
+echo "################################"
+
+velocitas create $VELOCITAS_CREATE_VALUES
```

Comment:
Call it args/arguments?

```suggestion
velocitas create $VELOCITAS_CREATE_ARGS
```

Reply:
done